keps/sig-multicluster/2149-clusterid/README.md

```diff
@@ -341,68 +419,134 @@ Contains a unique identifier for the containing cluster.
 
 *   The identifier **must** either:
     *   be a valid [RFC-1123](https://tools.ietf.org/html/rfc1123) DNS label,
-    *   or be composed of two valid [RFC-1123](https://tools.ietf.org/html/rfc1123) DNS labels separated with a dot.
+    *   or be composed of two valid
```

Comment:
Leaning on two feels smelly.  Could we not define it as a fragment of a DNS subdomain, with the caveat that it needs to be composed into larger DNS names and still fit inside 253 ?

Reply:
I agree about the smell. Defining as a fragment makes more sense at this point. The caveat seems like it's going to cause pain at some point, but two valid labels certainly could as well. I don't see much loss here